db/migrations/9_create_subscriptions.sql

```diff
@@ -0,0 +1,46 @@
+CREATE TABLE public.subscriptions_set
+(
+    id                     uuid                                 DEFAULT public.gen_random_uuid() NOT NULL,
+    name                   character varying           NOT NULL,
+    state                  character varying           NOT NULL DEFAULT 'initial',
+    restarts_count         integer                     NOT NULL DEFAULT 0,
```

Comment:
The number of times the SubscriptionsSet's runner(or Subscription's runner in case with `subscriptions` table) was restarted

Reply:
> and restart_count it should be i think, the s is not needed

Why? it is not a single restart - there are many. In normal pronunciation it would be `Number of restarts`

Reply:
Just dont think it is required, because restart_count already implies multiple. Its like this guy here using "node_count", not nodes_count: https://stackoverflow.com/questions/548646/naming-a-count-variable 
So id say keep it simple, remove the s's in such cases and there are a few such cases in this pr. @aroeczek, what do you think ?

Reply:
And why do subscriptions need a runner and subscription set too?

Reply:
Alright, will rename

Reply:
> And why do subscriptions need a runner and subscription set too?

Subscriptions set runner is `SubscriptionFeeder` class which actually fetches events and feeds them to subscriptions